tensorflow_datasets/testing/dataset_builder_testing.py

```diff
@@ -261,6 +272,20 @@ def test_download_and_prepare_as_dataset(self):
     else:
       self._download_and_prepare_as_dataset(self.builder)
 
+    if not self.SKIP_CHECKSUMS:
+      with self._subTest("url_checksums"):
+        self._test_checksums()
+
+  def _test_checksums(self):
+    filepath = os.path.join(checksums._get_path(self.builder.name))
+    if tf.io.gfile.exists(filepath):
+      sizes_checksums = checksums._get_sizes_checksums(filepath)
+      urls = sizes_checksums.keys()
+    else:
+      raise AssertionError("url checksums file not found at %s" % filepath)
+
+    self.assertTrue(self._download_urls.issubset(set(urls)), "url checksums don't match.")
```

Comment:
I'm sure there is a `self.assertSubSet`, or a better function than `assertTrue `.

Reply:
I could only find [assertSetEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertSetEqual).
A similar subset function [assertDictContainsSubset](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertDictContainsSubset) is deprecated